src/core/containers/HandleLogin/index.js

```diff
@@ -33,7 +34,8 @@ class HandleLogin extends React.Component {
   }
 
   render() {
-    const { code, state } = this.props.location.query;
+    const { location } = this.props;
+    const { code, state } = location.query;
     const { error } = this.state;
```

Comment:
Yeah, I see what you're getting at now. We should probably dispatch an action for success and error to be consistent and it will make it easier for other components to depend on this. I'll leave it for the other issue though since this is working as is and we don't currently have a need for it in redux yet.